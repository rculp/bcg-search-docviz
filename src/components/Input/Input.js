import { Input } from 'semantic-ui-react';
import './Input.scss';

export default Input;
