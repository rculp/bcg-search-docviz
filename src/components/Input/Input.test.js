import { shallow } from 'enzyme';

import React from 'react';

import Input from './Input';

describe('Input', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Input />
    );
    expect(component).toMatchSnapshot();
  });
});