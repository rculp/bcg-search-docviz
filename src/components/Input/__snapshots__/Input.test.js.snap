// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders and matches our snapshot 1`] = `
<div
  className="ui input"
>
  <input
    onChange={[Function]}
    type="text"
  />
</div>
`;
